components/Trip/index.js

```diff
@@ -1,21 +1,18 @@
-import Image from "next/image";
-import Link from "next/link";
 import Location from "../Location";
 import TimePeriod from "../TimePeriod";
+import { StyledList, Wrapper, StyledLink, StyledImage } from "./Trip.styled";
 
 export default function Trip({ title, img, location, timePeriod, _id }) {
   return (
-    <Link href={`/trip/${_id}`}>
-      <li>
-        <Image src={img} width={150} height={100} alt="Dummy Pic" />
-        <br />
-        {title}
-        <br />
-        <Location location={location} />
-        <br />
-        <TimePeriod timePeriod={timePeriod} />
-        <br />
-      </li>
-    </Link>
+    <StyledLink href={`/trip/${_id}`}>
```

Comment:
That would make a huge link!
What if I want to select and to copy the location? :/
Make only the title and the image clickable

Reply:
As a team we have decided to make the whole list item clickable since we think that the user would expect the whole card to be clickable instead of only the image and title. If the user feels the need to copy the text, they can still do so in the trip details page. 😊